### ISSUE TITLE
False alarm

### ISSUE THREAD

**Issue by hongbo-miao** [editorconfig-checker/editorconfig-checker#517]
I think I found a weird issue. I have a file `a.json` with content

```
{
  "type": "Feature"
}
```

This will gives error

```sh
> ec
Wrong character encoding ("" instead of "utf-8")
```

However, if change for example F to G, then no more errors

```
{
  "type": "Geature"
}
```

Here is my `.editorconfig`

```
root = true

[*.json]
charset = utf-8
indent_size = 2
indent_style = space
end_of_line = lf
insert_final_newline = true
trim_trailing_whitespace = true
```

`.editorconfig-checker.json`

```json
{
  "Debug": false,
  "IgnoreDefaults": false,
  "NoColor": false,
  "SpacesAfterTabs": false,
  "Verbose": false,
  "Disable": {
    "Charset": false,
    "EndOfLine": false,
    "Indentation": false,
    "InsertFinalNewline": false,
    "TrimTrailingWhitespace": false,
    "IndentSize": false,
    "MaxLineLength": false
  }
}
```

Any ideas? Thanks! ☺️

**Comment by mstruebing** [comment]
@rasa as you've implemented that feature ... any idea what the issue might be?

**Comment by rasa** [comment]
> [@rasa](https://github.com/rasa) as you've implemented that feature ... any idea what the issue might be?

@mstruebing No, only:

@hongbo-miao Are you sure you are using 3.6.0? The 
```
Wrong character encoding ("" instead of "utf-8")
```
issue was in 3.5.0, but fixed in 3.6.0.

If you are sure you are using 3.6.0, can you provide a link to the file, and I can test on my end.
